net.launchpad.gmult.yaml

```diff
@@ -0,0 +1,75 @@
+# -*- Mode: YAML; indent-tabs-mode: nil; tab-width: 2 -*-
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Michael Terry
+---
+app-id: net.launchpad.gmult
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: gmult
+
+finish-args:
+  - --device=dri # gtk4 wants gl access
+  - --socket=fallback-x11
+  - --socket=wayland
```

Comment:
`--share=ipc` is still required

Reply:
Added! I wondered about that. Everything seemed to still be working fine in x11 without it, but the docs did say x11 would need it.